README.md

```diff
@@ -1,2 +1,181 @@
+| audit   | The "audit" account is where all logs end up.                                        |
+| corp    | The "corp" account is where you run platform services for the company                |
+| data    | The "data" account is where the quants live =)                                       |
+
+Each account has its own [terraform state backend](https://github.com/cloudposse/terraform-aws-tfstate-backend), along with a [dedicated DNS zone](https://www.terraform.io/docs/providers/aws/r/route53_zone.html) for service discovery.
+
+The root account owns the top-level DNS zone and then delegates NS authority to each child account.
+
+### Assumptions
+
+1. We are starting with a clean AWS environment and a new "root" (top-level) AWS account. This means you need the "root" credentials, since a fresh AWS account doesn't even have any AWS roles that can be assumed.
+2. You have administrator access to this account
+3. You have [docker](https://docs.cloudposse.com/tools/docker/) installed on your workstation
+4. You have [terraform](https://www.terraform.io/downloads.html) installed on your workstation
+
+### Checklist
+
+Before we get started, make sure you have the following
+
+- [ ] Before you can create new AWS accounts under your organization, you must [verify your email address](https://docs.aws.amazon.com/console/organizations/email-verification).
+- [ ] Open a support ticket to [request the limit](https://console.aws.amazon.com/support/v1#/case/create) of AWS accounts be increased for your organization (the default is 1).
+- [ ] Clone this repo on your workstation
+- [ ] Create a *temporary* pair of [Access Keys](https://console.aws.amazon.com/iam/home#/security_credential). These should be deleted afterwards.
+- [ ] Export your AWS "root" account credentials as `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` (this is *temporary* for bootstrapping)
+- [ ] An available domain we can use for DNS-base service discovery (E.g. `ourcompany.co`). This domain must not be in use else where.
+- [ ] Ensure that any users who will be added during this bootstrap process have setup their [keybase](http://keybase.io) profile. You'll need this if setting them up in the `users` section of the [`config/root.tfvars`](https://github.com/cloudposse/reference-architectures/blob/master/configs/root.tfvars). For example you should be able to [verify their public key](https://keybase.io/osterman/key.asc) on `keybase.io` by running `curl https://keybase.io/$username/key.asc`.
+
+### 1. Provision Root Account
+
+The "root" account is the top-most AWS account from which all other AWS accounts are programmatically created.
+
+__WARNING:__ Terraform cannot remove an AWS account from an organization. Terraform will not close the account. The member account must be prepared to be a standalone account beforehand. See the [AWS Organizations documentation](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_remove.html) for more information.
```

Comment:
Above it says `all other AWS accounts are programmatically created`, but then `The member account must be prepared to be a standalone account beforehand`. More info needed on why and how?